src/main/java/com/zendesk/maxwell/schema/MysqlSavedSchema.java

```diff
@@ -437,7 +457,8 @@ private void restoreTable(Connection connection, Database d, String name, int id
 					cRS.getString("name"), cRS.getString("charset"),
 					cRS.getString("coltype"), i++,
 					cRS.getInt("is_signed") == 1,
-					enumValues);
+					enumValues,
+					Long.valueOf(cRS.getInt("column_length")));
```

Comment:
this will always make `column_length` `0`.. not sure if you  were trying to allow NULL here or if you intended that "zero is a good enough null"